### PR TITLE
Fix/ckan check resource view list

### PR DESCRIPTION
**Pull request by AlvaroVega**
issue:  https://github.com/telefonicaid/fiware-cygnus/issues/2150

ckan doc about API resource_view_list
https://github.com/okfn/ckan-api-blueprint/blob/master/blueprint_src.md#resource-view-list-api3actionresource_view_list

### CODE REVIEW

**Comment by AlvaroVega** [comment]
The main change of this PR is the evaluation of the result, changing from >0 to >= 0 
And also logging result of evaluation, because before this PR some times is logging ok but returning non ok.

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1 +1 @@
-
+- [cygnus-common][CKANBackendImpl] allow empty result as as valid in existsView (resource_view_list) (#2150)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- [cygnus-common][CKANBackendImpl] allow empty result as as resource_view_list (#2150)
```

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1 +1 @@
-
+- [cygnus-common][CKANBackendImpl] allow empty result as as resource_view_list (#2150)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- [cygnus-common][CKANBackendImpl] allow empty result as resource_view_list (#2150)
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**